variables.tf

```diff
@@ -278,6 +278,11 @@ variable "install_playbook_tag" {
     default = "10fec74c9e987b39f7af1127abe304a9e41f8e65"
 }
 
+variable "helm_repo" {
+    description = "Set the URL after http_server_repo_main_dir pointing to the Python helm modules"
+    default = "https://get.helm.sh/helm-v3.4.0-linux-ppc64le.tar.gz"
```

Comment:
Since we are at it can you change this to the latest version? https://get.helm.sh/helm-v3.6.3-linux-ppc64le.tar.gz

Reply:
Have updated it with the mentioned version.